src/spineml/standalone_simulator/main.cc

```diff
@@ -0,0 +1,55 @@
+// SpineML simulator includes
+#include "simulator.h"
+
+using namespace SpineMLSimulator;
+
+int main(int argc, char *argv[])
+{
+    try
+    {
+        if(argc < 2) {
+            throw std::runtime_error("Expected experiment XML file passed as arguments");
+        }
+
+        // Read min log severity from command line
+        const plog::Severity minSeverity = (argc > 3) ? (plog::Severity)std::stoi(argv[3]) : plog::info;
+
+        // Initialise log channels, appending all to console
+        plog::ConsoleAppender<plog::TxtFormatter> consoleAppender;
+        plog::init(minSeverity, &consoleAppender);
+
+#ifdef _WIN32
+        // Startup WinSock 2
+        WSADATA wsaData;
+        if(WSAStartup(MAKEWORD(2, 2), &wsaData) != 0) {
+            throw std::runtime_error("WSAStartup failed");
+        }
+
+#endif  // _WIN32
```

Comment:
I think you also need to call ``WSACleanUp`` (or whatever it's called) before the program exits, or your socket can end up locked

Reply:
:+1: